iPetCare/API/Controllers/UserController.cs

```diff
@@ -63,5 +64,20 @@ public async Task<ActionResult<GetAllUsersDtoResponse>> GetUsers()
                 return Forbid(response.Message);
             return BadRequest(response.Message);
         }
+
+        [AuthorizeRoles(Role.Administrator, Role.Vet, Role.Owner)]
+        [HttpPut]
+        public async Task<ActionResult<EditProfileDtoResponse>> EditProfile(EditProfileDtoRequest dto)
+        {
+            var response = await _userService.EditProfileAsync(dto);
```

Comment:
A czemu tutaj zrezygnowaliśmy z korzystania z tego fajnego dwulinijkowca do obsługi kontrolerów? Tzn widze, że ogólnie ten kontroler u góry ma jakiś problem bo z tego nie korzysta kij wie czemu XD

Chyba na branchu domergowywanym tego nie było albo coś :CC

Albo kopmpletnie zapomniałem o jego refaktorze

Reply:
To chyba nie było refaktoryzowane wcale, więc póki co dostosowałam się do starej konwencji bez refaktoru

Reply:
To zostawić tak i zrobisz refaktor dla całego usera?

Reply:
Tak zrobi xD